internal/cmd/completion_command.go

```diff
@@ -118,7 +119,12 @@ func (c *CompletionCommand) Run(args []string) int {
 		return 1
 	}
 
-	schema, err := modMgr.SchemaForModule(fh.Dir())
+	var sc *schema.BodySchema
+	if file.LanguageID() == ilsp.Tfvars.String() {
+		sc, err = modMgr.SchemaForVariables(file.Dir())
+	} else {
+		sc, err = modMgr.SchemaForModule(file.Dir())
+	}
```

Comment:
I feel this logic should probably be extracted out to avoid repetition in the handlers; not sure where to put it, that's one of the reasons it was initially implemented in the module manager.

Reply:
re: completion command, I left it as it originally was since it doesn't have access to `schemaForDocument()` in `service.go` and since you're thinking of deprecating it...